packages/wagmi/package.json

```diff
@@ -26,9 +20,11 @@
             "default": "./_cjs/index.js"
         }
     },
+    "dependencies": {
```

Comment:
This is temporary as need to install a different version of permissionless locally, will need to port wagmi to new permissionless with 0.7 version too